slack_connector/main.py

```diff
@@ -106,6 +148,7 @@ def main():
     # Filter data
     df = filter_data(df)
     
+    # Create csv file
     df.to_csv(args.local_dir+'/'+args.file_name, index=False)
     
          
```

Comment:
in line 163 below the except:  statement is catching all exceptions.  Try to catch only the specific Exception.